src/utils/sources-tree/utils.js

```diff
@@ -34,7 +34,7 @@ export function isInvalidUrl(url: Object, source: Object) {
   return (
     IGNORED_URLS.indexOf(url) != -1 ||
     !source.get("url") ||
-    source.get("loading") ||
+    source.loadedState === "loading" ||
```

Comment:
heya @sharathnarayanph what error are you getting here? Seems to work for me! Was this what you used`!isLoaded(source.toJS())`?